### ISSUE TITLE
negative items on disk exporting importing with stack upgrade and 3 speed upgrades

### ISSUE THREAD

**Issue by ilordnightmarei** [raoulvdberge/refinedstorage#437]
#### Issue description: negative items on disk exporting with stack upgrade and 3 speed upgrades
and importer that is exporting with same configuration removing disks and putting them back in fixes this 

#### What happens: negative items on disk exporting with stack upgrade and 3 speed upgrades
and importer that is exporting with same configuration removing disks and putting them back in fixes this  also notied i think theres a mscalulation with stack upgrade as in  its  doing stacks and if its not a stack it puts a negative number or subtracted of that stack

#### Steps to reproduce:
1. http://imgur.com/a/oEUcc a setup like this will cause it
2. importer 3 speed upgrades 1 stack 
3.exporter  3 speed upgrades 1 stack
...inventory in the middle storage crates hold alot more than chest and work like chests in fact there just big chests

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge:2094
- Refined Storage:1.1.2

Does this issue occur on a server? [yes/no]
yes
#### If a (crash)log is relevant for this issue, link it here:

[pastebin/gist/etc link here]

**Comment by thiakil** [comment]
I have observed this too after bulk exporting my system contents - I'm not sure the Importer is relevant though as I was just exporting out of the drives. As mentioned, pulling a drive out and reinserting it refreshes the (assuming) cached value.

![2016-09-18_20 45 47](https://cloud.githubusercontent.com/assets/1788847/19210147/32481ec2-8d4d-11e6-94e6-a8198e099320.png)

I am unable to comment on whether items were duped or not as I did not take note of the number stored before export.
